soccer/gameplay/main.py

```diff
@@ -227,6 +228,7 @@ def run():
         if root_play() is not None:
             situationAnalysis.updateAnalysis()
             root_play().spin()
+        robot_viz.draw_robot_viz(our_robots(), their_robots(), debug_drawer())
```

Comment:
Personally, if this is just being drawn every frame, I think this fits better in the GUI. 

https://github.com/RoboJackets/robocup-software/blob/fa29951e7c3a3de608252b77511093a15ce0bbd8/soccer/ui/FieldView.cpp#L654

Reply:
ah ok, I guess that makes more sense.

Reply:
I would agree with Joe.